--- a/FreeFileSync/Source/Makefile
+++ b/FreeFileSync/Source/Makefile
@@ -15,7 +15,7 @@
 
 #################### Linux ############################
 ifeq ($(OPERATING_SYSTEM_NAME), Linux)
-COMPILER_BIN=g++ -pthread
+COMPILER_BIN=g++-4.7 -pthread
 CXXFLAGS += -DZEN_LINUX
 
 #Gtk - support recycler/icon loading/no button border/grid scrolling
--- a/FreeFileSync/Source/RealtimeSync/Makefile
+++ b/FreeFileSync/Source/RealtimeSync/Makefile
@@ -12,7 +12,7 @@
 
 #################### Linux ############################
 ifeq ($(OPERATING_SYSTEM_NAME), Linux)
-COMPILER_BIN=g++ -pthread
+COMPILER_BIN=g++-4.7 -pthread
 CXXFLAGS += -DZEN_LINUX
 
 #Gtk - support "no button border"
